README.md

```diff
@@ -131,6 +131,11 @@ fmo quantize \
 --offload
 ```
```

Comment:
```
export MODEL_NAME_OR_PATH="meta-llama/Meta-Llama-3.1-8B-Instruct"
export OUTPUT_DIR="./"
export QUANTIZATION_SCHEME=fp8
export PEDANTIC_LEVEL=1
export DEVICE=1

fmo quantize \
--model-name-or-path $MODEL_NAME_OR_PATH \
--output-dir $OUTPUT_DIR \
--mode $QUANTIZATION_SCHEME \
--pedantic-level $PEDANTIC_LEVEL \
--device $DEVICE \
--offload
```
Is it better to align with the screenshot?

Reply:
Sure, I applied the modification.